lib/routes/requests.js

```diff
@@ -0,0 +1,97 @@
+var Boom = require('boom')
+
+/* NOTES:
+  * What Auth do we use against CouchDB to get/set user doc? (user not logged in)
+  * What url do we send to the user? (how do configure url)
```

Comment:
The origin should be in `server.info.uri`. See also the `notifications.templates.passwordreset` option. I think that should work, right?